docs/user/resources/04-100-10-aws-nfs-volume-restore.md

```diff
@@ -0,0 +1,51 @@
+# AwsNfsVolumeRestore Custom Resource
+
+The `awsnfsvolumerestore.cloud-resources.kyma-project.io` namespaced custom resource (CR) describes the AWS EFS file 
+system full restore operation on the same existing EFS file system.
+
+Out-of-place restore and Item-Level restore are not supported at this version of the `Cloud Manager`.
+
+While the EFS file system restore operation is running in the underlying cloud provider subscription, it needs its 
+source backup and its destination file system to be available. Upon its completion, restored file system 
+would be available on a recovery directory, named aws-backup-restore_{datetime}, off of the root directory. 
+
+## Specification <!-- {docsify-ignore} -->
+This table lists the parameters of the given resource together with their descriptions:
```

Comment:
This sentence adds zero value

Reply:
> This sentence adds zero value

This sentence appears in all other resource documents in cloud-manager repo. So I have it here for consistency. If you think it needs to be removed, I suggest a new issue is created so we remove it from all documents to be consistent. What do you think?

Reply:
No worries, keep it for consistency. No need for an extra issue.